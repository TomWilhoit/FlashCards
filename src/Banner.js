import React, { Component } from "react";
import "./css/Banner.scss";

class Banner extends Component {
  render() {
    return (
      <div className="banner">
        <h1>FEE MOD2 Flashcards</h1>
      </div>
    );
  }
}

export default Banner;
