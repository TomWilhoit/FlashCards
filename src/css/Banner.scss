.banner {
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: red;
  height: 100px;
  font-size: 25px;
  margin: 25px;
}