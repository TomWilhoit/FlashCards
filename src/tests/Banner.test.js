import React from 'react';
import Banner from '../Banner';
import { shallow } from 'enzyme';


describe('Banner', () => {
  
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Banner/>
      )
    });
    it('should match snapshot when all data is passed correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  })
