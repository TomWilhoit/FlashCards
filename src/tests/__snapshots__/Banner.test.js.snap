// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should match snapshot when all data is passed correctly 1`] = `
<div
  className="Banner"
>
  <h1>
    FEE MOD2 Flashcards
  </h1>
</div>
`;
